source/chapters/2/intro.md

```diff
@@ -6,45 +6,43 @@ disqusPage: 'Chapter 2: Intro'
 
-Meteor is a data-system more than anything else, ofcourse it gives you some snacks to help you
-code faster: The User System, Emails, much more. But the View Layer it's up to you.
+Since Meteor is a system that focuses on data more than anything else, of course it gives you some snacks to help you
+code faster: The User System, Emails and so much more. But when it comes to the View Layer, it's up to you!
 
-Don't get this the wrong way, Meteor can be integrated with any View Layer/Template Engine/UI Framework you want to use. 
+Don't panic! Meteor can be integrated with any View Layer/Template Engine/UI Framework you want to use! 
 
 ## Brief history
 
-When Meteor started there was no stable solution for building reactive views. They had to invent a way.
-This is how Blaze was born. [http://blazejs.org/](http://blazejs.org/). In Q4 2015, they decided to drop
+When Meteor started there weren't any stable solutions for building reactive views with it. Because of that the developers had to invent a way.
+This is how [Blaze](http://blazejs.org/) was born. In Q4 of 2015, the developers decided to drop the 
 support for Blaze, not because it wasn't good, but because there were other teams that focused only on the
 view layer. You cannot expect one company to do everything. So they began decoupling Blaze from Meteor.
 
-Even if Blaze is one of the most friendliest View Layers you'll ever meet, they gave it to the community
-to take care of it, decoupling it from Meteor. This does not mean you should not use Blaze. Blaze is rock-solid and battle-tested.
+The Meteor developers gave Blaze to the community to take care of it, decoupling it from Meteor. 
 
-However, from a personal opinionated experience, we noticed that for mobile builds, Blaze is inferior to React, in terms
+Even if it was a component of Meteor for a while, we do not recommend using Blaze in your applications.
+From personal experiences, we noticed that, for mobile builds, Blaze is inferior to React in terms
 of speed and stability. Especially for older mobile phones.
 
-However, times change, new View Layers appeared that support reactivity such as:
-- React
-- Vue.JS
-- Angular2
-- etc
+However, as times change, new View Layers appeared that support reactivity:
+- [React](https://reactjs.org/)
+- [Vue.JS](https://vuejs.org/)
+- [Angular2](https://angularjs.org/)
 
-Each has advantages and disadvantages. So question now is what to choose ?
+Each of these has advantages and disadvantages. The question is: what should you choose?
 
-Our current recommendation is: [React](https://facebook.github.io/react/)
+Our recommendation is [React](https://facebook.github.io/react/)
 
-Reasons for this choice:
+The reasons for which we made this choice are:
 
-- Modular
-- A bit hard to use at first, but much easier once you get the hang of it, it becomes easy and intuitive
-- Lightning Fast 
-- Maintained by Facebook
```

Comment:
maybe we should say about facebook as well, some devs might now know that (when they are juniors), it gives more confidence and stability